### ISSUE TITLE
md-icon-button: Icon not centered

### ISSUE THREAD

**Issue by bluestella** [angular/material#7441]
Needs to override below css style to line-height 0px for the icon to be centered.
```

.md-button.md-icon-button {
  width: 39px !important;
  height: 39px !important;
**  line-height: 0px !important;**
}
```

**Comment by ivoviz** [comment]
Related #7257

**Comment by DevVersion** [comment]
@ivoviz Can you update your issue to be more generic, then we can go with yours, instead of having two.

**Comment by ivoviz** [comment]
@DevVersion Sure thing, will do it in a bit.

**Comment by DevVersion** [comment]
@bluestella I'll close this issue, because it's caused by the same origin as in #7257. 
In addition to that - @ivoviz updated his description to track your issue too.

So thanks for the issue and *fix suggestion*- Please keep track of #7257
